kernel/kernel-impl/src/main/sql/oracle/sakai_config_item.sql

```diff
@@ -27,12 +27,4 @@ ALTER TABLE SAKAI_CONFIG_ITEM
 CREATE INDEX SCI_NODE_IDX ON SAKAI_CONFIG_ITEM (NODE ASC);
 CREATE INDEX SCI_NAME_IDX ON SAKAI_CONFIG_ITEM (NAME ASC);
 
-CREATE SEQUENCE SAKAI_CFG_ITEM_SEQ;
-
-CREATE OR REPLACE TRIGGER SCI_ID_AI
-BEFORE INSERT ON SAKAI_CONFIG_ITEM
-FOR EACH ROW
-BEGIN
-  SELECT SAKAI_CFG_ITEM_SEQ.NEXTVAL INTO :new.ID FROM dual;
-END;
-/
+CREATE SEQUENCE SAKAI_CONFIG_ITEM_S;
```

Comment:
Is this sequence manually used in code? Because that trigger was assigning the ID.